### ISSUE TITLE
EXC_BAD_ACCESS

### ISSUE THREAD

**Issue by tommyle** [philackm/ScrollableGraphView#96]
I'm getting EXC_BAD_ACCESS when trying to run the example code with CocoaPods. If I run the example with the included source files it works fine but when I use cocoapods I get a EXC_BAD_ACCESS when setting the graphView.dataSource. Is this an issue with 4.0.4?

<img width="896" alt="screen shot 2017-07-25 at 1 55 54 pm" src="https://user-images.githubusercontent.com/656538/28586281-3c6144c8-7141-11e7-9276-ad9c6b6056fb.png">

**Comment by tommyle** [comment]
When using cocoapods you need to set the Module to ScrollableGraphView
<img width="331" alt="screen shot 2017-07-25 at 4 26 36 pm" src="https://user-images.githubusercontent.com/656538/28592165-21954828-7156-11e7-86c7-47901558749a.png">

**Comment by AbidSana** [comment]
I have the same problem. Did you ever find a way to resolve this?

**Comment by tommyle** [comment]
@AbidSana Yes, please see the screen shot i posted. You need to set the Module to ScrollableGraphView.

**Comment by AbidSana** [comment]
Yes, I set the Module to ScrollableGraphView like in your screen shot, but the exception still persists.
